packages/opencensus-instrumentation-http/src/http.ts

```diff
@@ -209,6 +211,8 @@ export class HttpPlugin extends BasePlugin {
                 'header with "x-opencensus-outgoing-request" - do not trace');
             return original.apply(this, arguments);
           }
+
+          pathname = url.parse(options.path).pathname;
```

Comment:
You might want to do `options.pathname` || `url.parse(options.path).pathname`. [According to Node docs][1], the options parameter can be a WHATWG URL object, which [won't have a `path` field but will have `pathname`][2].

[1]: https://nodejs.org/api/http.html#http_http_request_options_callback
[2]: https://nodejs.org/api/url.html#url_the_whatwg_url_api